shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -160,6 +160,21 @@
   frame_size_ = frame_size;
 }
 
+bool FlutterPlatformViewsController::HaveEmbeddedViewsMutated() {
+  if (!views_to_recomposite_.empty()) {
+    return true;
+  }
+  if (active_composition_order_.size() != composition_order_.size()) {
```

Comment:
nit: can we just simply compare `if (active_composition_order_ != composition_order_) return true;`  I might be missing something in c++

Reply:
Done